web/src/settings_org.ts

```diff
@@ -209,12 +230,20 @@ function update_message_edit_sub_settings(is_checked) {
     );
 }
 
-function set_msg_edit_limit_dropdown() {
+function set_msg_edit_limit_dropdown(): void {
     settings_components.set_time_limit_setting("realm_message_content_edit_limit_seconds");
 }
 
-function message_move_limit_setting_enabled(related_setting_name) {
-    const setting_value = Number.parseInt($(`#id_${CSS.escape(related_setting_name)}`).val(), 10);
+function message_move_limit_setting_enabled(related_setting_name: string): boolean {
```

Comment:
zod parse as one of:

* realm_edit_topic_policy
* realm_move_messages_between_streams_policy

then assert the value is a number